### DISCUSSION TITLE
Goodbye

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#948] [General]
I will revisit this project when you have C++17 and 64 bit support.

**Comment by LADSoft** [comment]
that would be nice :smile:    

well the plan right now is finish working with gnu cobol which I'm allotting a bounded period of about two weeks for, that should be ample time for the current work.   Then finish C23 support which should take 2-4 weeks, then finish C17 support which should take 2-3 months, then do the x64 support which by the current plan is likely to take six to 8 months; beyond the C17 support none of the work is very complex just lots of details....   

Orange C should become version 7.0 when C17 support is released.

I am thinking though about making a change to the approach to x64 support where i can get it done more expeditiously.    What I really wanted to do is support the C/C++ language in the ADL compiler, but, maybe that is reaching too far for the present needs and I can circle back to that after we get some of the other work done...    that would cut several months off getting x64 support out the door I think as we would be limiting the scope to modifying the compiler/linker/downloader and updating the RTL.

well i guess I will go adjust the milestones lol...
